src/api/app/models/bs_request.rb

```diff
@@ -506,6 +506,15 @@ def permission_check_change_state!(opts)
     check_bs_request_actions!(skip_source: true)
   end
 
+  def permission_check_change_state(opts)
+    begin
+      permission_check_change_state!(opts)
+    rescue PostRequestNoPermission
```

Comment:
There are possibly more:

- BsRequest::Errors::CreatorCannotAcceptOwnRequests
- PostRequestMissingParameter

Not sure if we should catch them, sounds confusing to me...

Reply:
or we catch in the policy directly...

Reply:
I decided against catching those "validations" in the permission check.